internal/internal_worker.go

```diff
@@ -768,6 +768,22 @@ func (r *registry) getWorkflowDefinition(wt WorkflowType) (WorkflowDefinition, e
 	return newSyncWorkflowDefinition(executor), nil
 }
 
+func (r *registry) getNexusService(service string) *nexus.Service {
+	r.Lock()
+	defer r.Unlock()
+	return r.nexusServices[service]
+}
+
+func (r *registry) getRegisteredNexusServices() []*nexus.Service {
```

Comment:
nit: Don't name Go functions `GetXXX` it's not idiomatic Go: https://go.dev/doc/effective_go#Getters.

Reply:
I was just following the conventions in the file: `getRegisteredActivities`, etc.